packages/node/src/configure/configure.module.ts

```diff
@@ -72,10 +72,7 @@ export class ConfigureModule {
         },
         {
           provide: SubqueryProject,
-          useValue: SubqueryProject.create(
-            loadProjectManifest(projectPath),
-            projectPath,
-          ),
+          useFactory: async () => SubqueryProject.create(projectPath),
```

Comment:
if path is not a valid tar file, the application will crash. We need to catch the error to make the reason of crash more clear.

Reply:
catch error added